pywinauto/application.py

```diff
@@ -839,6 +840,15 @@ def __connect(self, **kwargs):
     def connect(self, **kwargs):
         """Connects to an already running process"""
         connected = False
+
+        if 'timeout' in kwargs:

```

Comment:
If no timeout in the method call parameters, we need to use `app_start_timeout` by default.